specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/bigDataPool.json

```
@@ -570,6 +570,7 @@
           "description": "The minimum number of executors alloted",
           "x-ms-mutability": [
             "create",
+            "read",
             "update"
           ]
```

Rather than adding "read" you should just remove x-ms-mutability altogether, since [it's equivalent](https://github.com/Azure/autorest/tree/main/docs/extensions#x-ms-mutability:~:text=applying%20the%20extension%20as%20%22x%2Dms%2Dmutability%22%3A%20%5B%22create%22%2C%20%22read%22%2C%20%22update%22%5D%20or%20not%20applying%20it%20will%20have%20the%20same%20effect.) and less verbose.
```suggestion
          "description": "The minimum number of executors alloted"
```

---

> 

Could you share insights on resolving RoundTrip alerts https://vnext.s360.msftcloudes.com/blades/unifiedplatform?blade=KPI:cf751b97-2c95-420f-baa8-9a16e8643a5b~SLA:2~AssignedTo:All~Forums:All~_loc:UnifiedPlatform&def=&peopleBasedNodes=&tile=&global=4:c0ee70d5-102d-438c-8858-795b92dc0f99;rajanve_team;mahi_team

---

@sparatala I investigated one of the ROUNDTRIP_INCONSISTENT_PROPERTY alerts and I think this is likely a false positive.  I think this check was only recently added to Live Validation and might be flagging cases it shouldn't.  I've started a discussion with the ARM team on this.